[Docs] Add 9.0.8 release notes (#237398)
elastic/kibana: docs/release-notes/index.md

* Adjusts the `z-index` of the app menu header to not conflict with the Persistent Console [#224708]({{kib-pull}}224708).
* Fixes an issue preventing solution navigation submenu items from being displayed when the navigation is collapsed [#227705]({{kib-pull}}227705).

> ## 9.0.8 [kibana-9.1.4-release-notes]

Reviewer: ```suggestion
## 9.0.8 [kibana-9.0.8-release-notes]
```

% ::::{NOTE}
% ::::

### Fixes [kibana-9.0.8-fixes]

**Alerting**:
* Fixes text kerning issues in PDF/PNG exports of dashboards and visualizations [#235516]({{kib-pull}}235516).

**Dashboards and Visualizations**:

> * Fixes a bug where the dashboard title would not get updated in the breadcrumbs when edited from the list of dashboards [#236561]({{kib-pull}}236561).

Reviewer: ```suggestion
* Fixes an issue where the dashboard title would not get updated in the breadcrumb when edited from the list of dashboards [#236561]({{kib-pull}}236561).
```
